config/prod/verena-allen-chal.yaml

```diff
@@ -16,16 +16,16 @@ parameters:
   # Default account settings, leave alone unless you plan to override
   VpcName: "computevpc"
   KeyName: "scicomp"
-  AMIId: "ami-0b898040803850657"      # Amazon Linux 2 AMI 2.0.20190618 x86_64 HVM gp2
+  AMIId: "ami-0b898040803850657" # Amazon Linux 2 AMI 2.0.20190618 x86_64 HVM gp2
   # AMIId: "ami-0a313d6098716f372"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
 
   # Settings have default values but can be overriden. Set the below
   # parameters *only* if you want to override the defaults.
 
   # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
-  InstanceType: "t3.2xlarge"
+  InstanceType: "t3.small"
   # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
-  VolumeSize: "200"
+  # VolumeSize: "200"
```

Comment:
Leave this at 200 or it will wipe your instance.

Reply:
okay, thanks!